Slim/Plugin/ExtendedBrowseModes/strings.txt

```diff
@@ -279,3 +279,28 @@ PLUGIN_EXTENDED_BROWSEMODES_AUTHORS
 	FR	Auteurs
 	NL	Schrijvers
 	SV	Författare
+
+PLUGIN_EXTENDED_BROWSEMODES_TAGS
+	EN	User Defined Roles
+	DE	Benutzerdefinierte Rollen
+	FR	Rôles définis par l'utilisateur
+	ES	Roles definidos por el usuario
+	NL	Door de gebruiker gedefinieerde rollen
+
+PLUGIN_EXTENDED_BROWSEMODES_TAGS_DESC
+	EN	Tell LMS about any custom contributor role tags used in the music library. For example, SOLOIST, DRUMMER, PRODUCER, etc. These can then be used as Contributor Roles in the Additional Browse Modes above. They will also appear as categories when browsing albums or artists with Release Type grouping enabled. Changes here will trigger a rescan of the music library.
```

Comment:
deepl.com does a decent job translating to German: https://www.deepl.com/de/translator#en/de/Tell%20LMS%20about%20any%20custom%20contributor%20role%20tags%20used%20in%20the%20music%20library.%20For%20example%2C%20SOLOIST%2C%20DRUMMER%2C%20PRODUCER%2C%20etc.%20These%20can%20then%20be%20used%20as%20Contributor%20Roles%20in%20the%20Additional%20Browse%20Modes%20above.%20They%20will%20also%20appear%20as%20categories%20when%20browsing%20albums%20or%20artists%20with%20Release%20Type%20grouping%20enabled.%20Changes%20here%20will%20trigger%20a%20rescan%20of%20the%20music%20library.